wgsl/index.bs

```diff
@@ -3692,20 +3692,20 @@ The construct is optional.
 
 The block of statements must not contain a return or discard statement.
 
-### Return ### {#return-statement}
+### Return Statement ### {#return-statement}
 
 <pre class='def'>
 return_statement
   : RETURN short_circuit_or_expression?
 </pre>
 
-A `return` statement ends execution of the current function.
-If the function is an entry point, then the current shader invocation
+A <dfn noexport>return</dfn> statement ends execution of the current function.
+If the function is an [=entry point=], then the current shader invocation
 is terminated.
```

Comment:
This reads to me quite similar to "discard" with this "terminated" term. Could we say the "shader invocation is finished" instead? That would indicate a "successful" ending.

Reply:
Termination means something ends. It's not about success or failure.

The discard statement description is:

> The discard statement causes the current invocation to terminate and no entry point return value to be generated for the invocation.

The "and no entry point return value to be generated..." is *additional* to termination.

I'd take a suggestion to update the discard statement description to indicate that the fragment is removed from downstream processing in the graphics pipeline. But that's separate.

Reply:
I filed https://github.com/gpuweb/gpuweb/pull/1554 to emphasize that in the discard statement, throwing away the fragment is a distinct action from terminating the invocation.